apps/preview/nuxt/pages/showcases/FormFields/FormFieldsRequired.vue

```diff
@@ -0,0 +1,634 @@
+<template>
+  <div class="px-4">
+    <h1 class="mb-4 typography-headline-4 font-bold">Personal information</h1>
+    <form @submit.prevent="sendForm">
+      <label class="flex justify-between typography-text-base cursor-pointer mb-4 gap-2">
```

Comment:
Is the `typography-text-base` needed here? It's the default text configuration, isn't it?

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2787/commits/361d45dcea22beee7c7f07252163b9882dd2946c